src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -15,20 +15,21 @@ class FlexMicroform extends React.Component {
 
     window.microform = null;
     this.state = {
-      microformStatus: DEFAULT_STATUS,
+      captureKey: { microformStatus: DEFAULT_STATUS },
```

Comment:
Don't need this change, I think?

Reply:
This wan't the right way to do that, fixed now.